public/app/plugins/datasource/cloud-monitoring/datasource.ts

```diff
@@ -352,10 +351,14 @@ export default class CloudMonitoringDatasource extends DataSourceApi<CloudMonito
         groupBys: this.interpolateGroupBys(metricQuery.groupBys || [], scopedVars),
         view: metricQuery.view || 'FULL',
       },
-      sloQuery: this.interpolateProps(sloQuery, scopedVars),
+      sloQuery: sloQuery && this.interpolateProps(sloQuery, scopedVars),
     };
   }
 
+  interpolateVariablesInQueries(queries: CloudMonitoringQuery[], scopedVars: ScopedVars): CloudMonitoringQuery[] {
+    return queries.map(query => this.prepareTimeSeriesQuery(query, scopedVars));
```

Comment:
You have checked the queries length in other data sources but not here. Shouldn't you check it?